core/server/data/migrations/versions/4.3/04-add-stripe-prices-table.js

```diff
@@ -0,0 +1,16 @@
+const addTable = require('../../utils').addTable;
+
+module.exports = addTable('stripe_prices', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    stripe_price_id: {type: 'string', maxlength: 255, nullable: false, unique: true},
+    stripe_product_id: {type: 'string', maxlength: 255, nullable: false, unique: false, references: 'stripe_products.stripe_product_id'},
+    active: {type: 'boolean', nullable: false},
+    livemode: {type: 'boolean', nullable: false},
+    nickname: {type: 'string', maxlength: 50, nullable: true},
+    currency: {type: 'string', maxLength: 3, nullable: false},
+    amount: {type: 'integer', nullable: false},
```

Comment:
I wanted to check, there is `unit_amount` and `unit_amount_decimal` do you know which one we plan to use/store here?

Reply:
So I had checked, and it should be `unit_amount` in our case as that maps to what we have been using with plans so far, which is the unit amount in paise represented as whole integer, while the `unit_amount_decimal` is the same thing represented as string and allowing upto 12 decimal points.